src/homepageExperience/components/steps/Finish.tsx

```diff
@@ -129,11 +129,26 @@ export const Finish = (props: OwnProps) => {
               InfluxDB.
             </p>
           </ResourceCard>
+          {props.wizardEventName === 'cliWizard' && (
```

Comment:
this might be a good place to use the [ternary operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator) so you don't need to repeat the `props.wizardEventName ===` check. It would look something like:

```ts
{props.wizardEventName === 'cliWizard' ? (
  // cli parts here
  ) : (
  <SampleAppCard
    wizardEventName={props.wizardEventName}
    handleNextStepEvent={handleNextStepEvent}
  />
)}

Reply:
tots see what you mean here. But I think i have to have two separate checks because one is rendered inside the `FlexBox` and one isn't. The cli command resource card needs to render inside the flexbox while the sample card needs to be outside.